TACC/Core-Portal
# task/FP-1190: Unified search for FP-737

## Overview: ##
A single term is used to find a unique set of results against the `username`, `lastName`, and `email` fields in TAS.

## Related Jira tickets: ##

* [FP-1190](https://jira.tacc.utexas.edu/browse/FP-1190)
* [FP-737](https://jira.tacc.utexas.edu/browse/FP-737)

## Summary of Changes: ##
- unified search
- allow overriding of `fetchParams` in `fetchUtil`

## Testing Steps: ##
1. Search for a `franklin`
2. look at requests and see they add up to 12
3. See that there are only 11 listed in dropdown (no duplicates, a unique set)

## UI Photos:
<img width="527" alt="Screen Shot 2021-09-01 at 3 18 48 PM" src="https://user-images.githubusercontent.com/20326896/131739169-255056ac-1042-4435-ab39-a1cb6514a4c7.png">
<img width="528" alt="Screen Shot 2021-09-01 at 3 21 19 PM" src="https://user-images.githubusercontent.com/20326896/131739448-63d9316e-d457-492c-aae7-538c2502eab3.png">
